src/governance.rst

```diff
@@ -88,9 +88,8 @@ the time when a vote is called, but may never be less than 24 hrs.
 
 :Nominate new member of core: The proposer must provide
     a sufficient justification as to why the nominee should be welcomed
-    into core. It is necessary that all nominees are current and active
-    members of staged-recipes. Nominees to core must have been members
-    of staged-recipes for at least 1 month (4 weeks) prior to this vote.
+    into core. It is not necessary that all nominees are active
```

Comment:
Maybe we could state this more generically? 

eg
`Prior service to the community, including but not limited to: serving as a staged-recipes reviewer, working on critical conda-forge infrastructure, and helping to bridge disparate communities are an important part of the nomination process`.

Reply:
I guess that "a sufficient justification a" is better than listing all possible reasons to justify the invitation and then modifying it when we find an issue, like this one.